projects/plugins/backup/jetpack-backup.php

```diff
@@ -36,6 +36,9 @@
 // Constant definitions.
 define( 'JETPACK_BACKUP_PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
 define( 'JETPACK_BACKUP_PLUGIN_ROOT_FILE', __FILE__ );
+define( 'JETPACK_BACKUP_PLUGIN_SLUG', 'jetpack-backup' );
+define( 'JETPACK_BACKUP_PLUGIN_NAME', 'Jetpack Backup' );
```

Comment:
I wonder if this may be something we'll want to translate? Since this isn't just a brand name, maybe we'll want "Backup" to be translated? That may be something worth asking to the Marketing and / or the Global team I think.

Reply:
That constant is only used here currently:

https://github.com/Automattic/jetpack/blob/e9ef5840315d9dcc749f7e2d22c6a93f62f76b81/projects/plugins/backup/src/php/class-jetpack-backup.php#L32-L38

So maybe it's better to remove the constant in a future PR to avoid usage of the untranslated text.

Reply:
I agree with @jeherve that we need to find out whether the plugin name "Jetpack Backup" will be translated to other languages before doing anything.
We don't translate "Jetpack Boost" though, AFAIK.